Snowflake.Data.Tests/UnitTests/PutGetStageInfoTest.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
```

Comment:
missing copyright header on top of the file?

Reply:
done